components/navigation.html

```diff
@@ -0,0 +1,166 @@
+                  >Badge</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Card</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Image</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Input</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Snackbar</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Badge</a
+                >
+              </li>
+            </ul>
+          </div>
+
+          <div class="sidebar-heading">Common Utilities</div>
+          <hr />
+          <div class="link-container">
+            <ul>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Display Utilities</a
+                >
+              </li>
+              <li>
+                <a class="sidebar-list-item" href="../components/docPage.html"
+                  >Flex Utilities</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="main-body">
+        <!-- Navigation-intro -->
+        <div class="body-block">
+          <div class="block-heading">Navigation</div>
+          <hr />
+          <div class="block-desc">
+            Navigation is used by user to naviagte to diferent parts of website
+            and it is a very important part of any website. follow the below
+            code example to get a fluid navbar easily.
+          </div>
+        </div>
+
+        <div class="body-block">
+          <div class="block-heading">Basic Example</div>
+          <hr />
+          <div class="exapmle-container">
+            <div class="elm-container">
+              <div class="navbar-container">
+                <div class="navbar-left-container">
+                  <i class="material-icons">menu</i>
+                  <div class="navbar-logo">
+                  BrandName
+                  </div>
+                </div>
+                <div class="navbar-right-container">
+                    <img
+                      src="../assets/images/git-mark.png"
+                      class="git-logo"
+                      alt="github-logo"
+                  /></a>
+                </div>
+              </div>
+            </div>
+            <div class="code-container">
+              <pre><code class="html-plaintext">&lt;div class="navbar-container">
+&lt;div class="navbar-left-container">
+    &lt;i class="material-icons">menu&lt;/i>
```

Comment:
Can format this part else is good to merge 👍

Reply:
Thanks @tusharkandpal but in this part the code needs to be this way i am using <code> tag in html so any identation or formatting would affect the code preview .